### ISSUE TITLE
Menubar apps should not quit

### ISSUE THREAD

**Issue by fuzzy76** [toggl/toggldesktop#1776]
The normal behaviour for menubar apps is to not quit until explicitly chosen from the right click menu. I keep accidentaly quitting toggl with command-Q.

**Comment by IndrekV** [comment]
Cmd+Q is the built in shutdown command I don't think it makes sense to try to block it. Many menubar apps that have menubar icons act the same way. (skype for instance)

**Comment by fuzzy76** [comment]
The normal way of doing this is adding CMD-Q as a shortcut to the menu item for quit. But menubar apps doesn't have a menu on their windows. So I am assuming Toggl actually has done extra work in order to catch CMD-Q when running without a dock icon.

**Comment by IndrekV** [comment]
It should work as follows:

If dock icon is visible - cmd+Q will quit the app.
If dock icon is not visible - cmd+Q does nothing. App can be closed by selecting quit from the menubar menu

**Comment by Leland** [comment]
FWIW, don't agree with this (at all).

Every single one of the menubar applications I have running are quittable via <kbd>Cmd</kbd>-<kbd>Q</kbd> when focused: Dropbox, Harvest, Creative Cloud, Fantastical, CloudApp.

**Comment by IndrekV** [comment]
Oh no @Leland you did not just bring Harvest into the play! On a more serious note. I've looked into the way many of the menubar apps work. And what I've found is that when the menubar app has no window to show it works as described by the creator of the issue and does not close the app with Cmd+Q. It makes sense as with these types of apps the app is never actually in focus as there is no window to focus. 

Apps that have window and menubar icon usually follow the rule that Cmd+Q will quit the whole app. With this new info, I revert the previous decision and we will not change the current shortcut logic for quitting Toggl Desktop.

Thank you for your input on this!
